src/gs-details-page.c

```diff
@@ -267,9 +266,6 @@ gs_details_page_update_copy_button (GsDetailsPage *self)
 	gtk_widget_set_sensitive (self->button_copy, FALSE);
 	gtk_widget_set_visible (self->button_copy, FALSE);
 
-	if (self->plugin_status == GS_PLUGIN_STATUS_COPYING)
```

Comment:
I don't think you want to remove this, since it would be wrong to update the copy button when the app is copying. Don't you want to change it to `gs_plugin_loader_app_copying()`?